config/application.rb

```diff
@@ -42,5 +42,11 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :sidekiq
 
     config.exceptions_app = routes
+
+    # Fetch Notify API Key if configured or raise exception with descriptive message
```

Comment:
I think the code is descriptive on its own without the comment:

```suggestion
```

Reply:
Comment removed